spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.feature '<<Pictures>>' do
+
+  before do
+    sign_up_1
+  end
+
+  context 'when no pictures have been added' do
+    scenario 'it should show prompt to upload an image' do
+      visit '/pictures'
+      expect(page).to have_content 'There aren\'t any pictures here yet.'
+      expect(page).to have_link 'Upload a picture...'
+      end
```

Comment:
Lint/BlockAlignment: `end` at 14, 6 is not aligned with `scenario 'it should show prompt to upload an image' do` at 10, 4.